packages/react-examples/src/react-components/Migrations/utils.test.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { createRenderer } from 'react-test-renderer/shallow';
+import { CodeExample } from './utils';
+
+it('renders children', () => {
+  const renderer = createRenderer();
+  renderer.render(
+    <CodeExample>
+      <span>Child</span>
+    </CodeExample>,
+  );
+  const result = renderer.getRenderOutput();
+
+  expect(result.props).toEqual({
+    children: 'Child',
+  });
+});
+
+it('renders Markdown source blocks', () => {
+  const renderer = createRenderer();
+  renderer.render(
+    <CodeExample>
+      {
+        {
+          props: {
```

Comment:
**q:** is this low level definition necessary?

Reply:
I'm afraid so as I couldn't find the exact component the Markdown source code block translates to in MDX 😟